test-clients/src/main/java/com/hedera/services/bdd/suites/E2EGlobalRunner.java

```diff
@@ -0,0 +1,482 @@
+import com.hedera.services.bdd.suites.records.DuplicateManagementTest;
+import com.hedera.services.bdd.suites.records.FeeItemization;
+import com.hedera.services.bdd.suites.records.FileRecordsSanityCheckSuite;
+import com.hedera.services.bdd.suites.records.MigrationValidation;
+import com.hedera.services.bdd.suites.records.MigrationValidationPostSteps;
+import com.hedera.services.bdd.suites.records.MigrationValidationPreSteps;
+import com.hedera.services.bdd.suites.records.RecordCreationSuite;
+import com.hedera.services.bdd.suites.records.RecordPurgeSuite;
+import com.hedera.services.bdd.suites.records.SignedTransactionBytesRecordsSuite;
+import com.hedera.services.bdd.suites.regression.AddWellKnownEntities;
+import com.hedera.services.bdd.suites.regression.JrsRestartTestTemplate;
+import com.hedera.services.bdd.suites.regression.SteadyStateThrottlingCheck;
+import com.hedera.services.bdd.suites.regression.UmbrellaRedux;
+import com.hedera.services.bdd.suites.regression.UmbrellaReduxWithCustomNodes;
+import com.hedera.services.bdd.suites.schedule.ScheduleCreateSpecs;
+import com.hedera.services.bdd.suites.schedule.ScheduleDeleteSpecs;
+import com.hedera.services.bdd.suites.schedule.ScheduleExecutionSpecStateful;
+import com.hedera.services.bdd.suites.schedule.ScheduleExecutionSpecs;
+import com.hedera.services.bdd.suites.schedule.ScheduleRecordSpecs;
+import com.hedera.services.bdd.suites.schedule.ScheduleSignSpecs;
+import com.hedera.services.bdd.suites.streaming.RecordStreamValidation;
+import com.hedera.services.bdd.suites.streaming.RunTransfers;
+import com.hedera.services.bdd.suites.throttling.PrivilegedOpsSuite;
+import com.hedera.services.bdd.suites.throttling.ThrottleDefValidationSuite;
+import com.hedera.services.bdd.suites.token.Hip17UnhappyAccountsSuite;
+import com.hedera.services.bdd.suites.token.Hip17UnhappyTokensSuite;
+import com.hedera.services.bdd.suites.token.TokenAssociationSpecs;
+import com.hedera.services.bdd.suites.token.TokenCreateSpecs;
+import com.hedera.services.bdd.suites.token.TokenDeleteSpecs;
+import com.hedera.services.bdd.suites.token.TokenFeeScheduleUpdateSpecs;
+import com.hedera.services.bdd.suites.token.TokenManagementSpecs;
+import com.hedera.services.bdd.suites.token.TokenManagementSpecsStateful;
+import com.hedera.services.bdd.suites.token.TokenMiscOps;
+import com.hedera.services.bdd.suites.token.TokenPauseSpecs;
+import com.hedera.services.bdd.suites.token.TokenTotalSupplyAfterMintBurnWipeSuite;
+import com.hedera.services.bdd.suites.token.TokenTransactSpecs;
+import com.hedera.services.bdd.suites.token.TokenUpdateSpecs;
+import com.hedera.services.bdd.suites.token.UniqueTokenManagementSpecs;
+
+// TODO: Refactor to better name
+// TODO: Remove stress tests if confirmed
+// TODO: Include performance tests if confirmed
+
+// TODO: Verify in daily to run sync() or async()
+// TODO: Implement a solution, extracting only failing test as console output
```

Comment:
Is this implemented yet?

Reply:
I'm waiting for @CordonaCodeCraft to finish his work on the maven profile. Those should have already been removed by that time.